src/main/java/me/dinowernli/grpc/prometheus/Configuration.java

```diff
@@ -77,7 +89,25 @@ public Configuration withLabelHeaders(List<String> headers) {
     List<String> newHeaders = new ArrayList<>(labelHeaders);
     newHeaders.addAll(headers);
     return new Configuration(
-        isIncludeLatencyHistograms, collectorRegistry, latencyBuckets, newHeaders);
+        isIncludeLatencyHistograms,
+        collectorRegistry,
+        latencyBuckets,
+        newHeaders,
+        addCodeLabelToHistograms);
+  }
+
+  /**
+   * Returns a copy {@link Configuration} with the difference that status code label will be added
+   * to latency histogram. If latency histogram itself is disabled, this takes no effect.
```

Comment:
Can we add a warning here that this will increase the number of histograms by a factor of the code cardinality, which could lead to using a lot of memory.

Reply:
Done, please check the wording)